src/isar_exr/api/energy_robotics_api.py

```diff
@@ -66,3 +68,42 @@ def pause_current_mission(self, exr_robot_id: str) -> None:
         ]
         if not success:
             raise RobotException(f"Invalid status after pausing mission: '{status}'")
+
+    def wake_up_robot(self, exr_robot_id: str) -> None:
+        query_string: str = """
+            mutation wakeUp($robot_id: String!)
+            {
+                executeAwakeCommand(targetState: AWAKE, $robot_id: String!)
+                {
+                    id
+                }
+            }
+        """
+        params: dict = {"robot_id": exr_robot_id}
+        try:
+            result: Dict[str, Any] = self.client.query(query_string, params)
+        except Exception as e:
+            raise RobotException(e)
+
+        while not self.get_robot_awake_status(exr_robot_id):
+            sleep(1)
```

Comment:
We should probably have a max wait time limit in the config.
After the max wakeup time, we fail assuming the robot cannot wake up. Maybe it is then dead